ReactAndroid/src/main/java/com/facebook/react/views/view/ReactViewBackgroundDrawable.java

```diff
@@ -442,6 +442,35 @@ private void drawRoundedBackgroundWithBorders(Canvas canvas) {
           }
         }
 
+        // colorInlineStart and colorInlineEnd  have precedence over colorEnd and colorStart values
```

Comment:
This function is already pretty chunky. Can we cleanly pull out the logic to resolve these values?